examples/factors/README.md

```diff
@@ -57,8 +55,8 @@ Don't worry about why these lines are included yet; for now, we're just being di
 Here are what the other two lines with `METHOD_NAME_ELF` and `METHOD_NAME_ID` should look like after updating:
 
 ```rust
-    let mut prover =
-        Prover::new(MULTIPLY_ELF).expect("Prover should be constructed from valid ELF binary");
+    let mut env =
+        ExecutorEnv::new(MULTIPLY_ELF).expect("Prover should be constructed from valid ELF binary");
```

Comment:
```suggestion
        ExecutorEnv::new(MULTIPLY_ELF).expect("ExecutorEnv should be constructed from valid ELF binary");
```

Reply:
Thanks! I clearly didn't really turn my brain on for this part of the review.

So yeah, suggestion is instead use the code that actually works from the factors example and describe it here.

Reply:
I can make the correct update and check the rest of the factors README for strays as well (MULTIPLY_ELF along with env as params for Executor construction).